.github/workflows/changelog.yml

```diff
@@ -0,0 +1,19 @@
+name: Changelog
+on: [workflow_dispatch, release]
+
+jobs:
+  changelog:
+    name: Update Changelog
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-ruby@v1
+      - run: gem install github_changelog_generator
+      - run: github_changelog_generator -u w3c -p "webidl2.js" --no-unreleased
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - uses: peter-evans/create-pull-request@v3
+        with:
+          commit-message: "chore(CHANGELOG): regenerate"
+          title: "chore(CHANGELOG): regenerate"
+          branch: regenerate-changelog
```

Comment:
@saschanaz you might also want to use the auto delete branch on merge/close option  (or enable it in GitHub repo settings)

Reply:
>or enable it in GitHub repo settings

This sounds better, thanks!